modules/apps/headless/headless-workflow/headless-workflow-impl/src/main/java/com/liferay/headless/workflow/internal/mutation/v1_0/Mutation.java

```diff
@@ -17,12 +17,6 @@
 import com.liferay.headless.workflow.dto.v1_0.WorkflowTask;
 import com.liferay.headless.workflow.resource.v1_0.WorkflowTaskResource;
 
-import graphql.annotations.annotationTypes.GraphQLField;
-import graphql.annotations.annotationTypes.GraphQLInvokeDetached;
-import graphql.annotations.annotationTypes.GraphQLName;
```

Comment:
So just to write it down. Keep the graphql.annotations.annotationTypes usages but remove the DataFetchingEnvironment parameters.

Reply:
So not what this commit is doing.